demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -92,32 +90,17 @@ function ProductsGridToolbar({ toolbarAction, exportApi }: { toolbarAction?: Rea
                 <GridFilterButton />
             </ToolbarItem>
             <ToolbarFillSpace />
-            <ToolbarItem>
-                <Button
-                    color="info"
-                    ref={moreActionsMenuRef}
-                    onClick={() => {
-                        setShowMoreActionsMenu(true);
-                    }}
-                    endIcon={<MoreVertical />}
-                >
-                    <FormattedMessage id="product.moreActions" defaultMessage="More actions" />
-                </Button>
-            </ToolbarItem>
+            <CrudMoreActionsMenu
+                overallActions={[
+                    {
+                        label: <FormattedMessage id="product.downloadAsExcel" defaultMessage="Download as Excel" />,
```

Comment:
Should this be a generic message in [messages.ts](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/messages.ts)?

Reply:
Good point, didn't know we had this but makes sense 😅